docs/content/DESeq2.md

```diff
@@ -50,65 +56,83 @@ For example:
 | sample_3 | hg38     | wildtype   | 2      | treatmentA |
 | sample_4 | hg38     | knockout   | 2      | treatmentA |
 | sample_5 | hg38     |            | 3      | treatmentB |
-| sample_6 | hg38     | unused     | 3      | treatmentB |
+| sample_6 | hg38     |            | 3      | treatmentB |
 
 ###### Contrast in the config.yaml
 Next, in the `config.yaml` add one or more contrasts:
 
 In order to compare two groups, the contrast condition is the column name, followed by the target group and finally the reference group.
-For example, to compare stage 1 to stage 3 from the examples above, the contrast would be `stages_3_1`.
-
-To compare all groups against one reference, the contrast condition is the column name, followed by target group "all" and finally the reference group.
-For example, to compare all treatments to the control from the examples above, the contrast would be `treatments_all_control`
-(our gremlins will carefully unpack all of your input and pass only unique designs to DESeq2).
+For example: to determine which genes/peaks are differentially expressed/active at stage 3 compared to stage 1, the contrast would be `stages_3_1`.
 
 ```
-# contrasts for examples 1-3 in the config.yaml
 contrasts:
-  - 'conditions_knockout_wildtype'
-  - 'stages_3_1'
-  - 'treatments_all_control'
+  - stages_3_1
+  - conditions_knockout_wildtype
+  - treatments_all_control        # the 'all' keyword is explained below
 ```
 
-###### Group order
-As a rule of thumb: the reference group should be specified **last**.
+The contrast `stages_3_1` will compare the expression of the target group (`3`, containing sample_5 and sample_6) against the reference group (`1`, containing sample_1 and sample_2).
+
+The contrast `conditions_knockout_wildtype` will compare the expression of the target group (`knockout`, containing sample_2 and sample_4) against the reference group (`wildtype`, containing sample_1 and sample_3).
+
+###### Count dispersions
+For each design contrast, you can specify which samples should be used to estimate the count dispersions by giving them a label.
+This can be used to exclude failed samples from the analysis.
+In general, more samples improves the dispersion estimate.
+
+In contrast `stages_3_1` all samples will be used to estimate the count dispersions, including sample_3 and sample_4.
+
+In contrast `conditions_knockout_wildtype` sample_5 and sample_6 will **not** be used to estimate the count dispersions.
```

Comment:
Sample 5 and 6 are not taken along because they are empty right? Perhaps explain that clearer.